schema/spacewalk/common/tables/suseContentProjectSource.sql

```diff
@@ -0,0 +1,46 @@
+--
+-- Copyright (c) 2018 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseContentProjectSource(
+    id NUMBER NOT NULL
+        CONSTRAINT suse_ct_prj_src_id_pk PRIMARY KEY,
+
+    type VARCHAR2(16) NOT NULL,
+
+    project_id NUMBER NOT NULL
+        CONSTRAINT suse_ct_prj_src_prjid_fk
+            REFERENCES suseContentProject(id)
+            ON DELETE CASCADE,
+
+    channel_id NUMBER
+        CONSTRAINT suse_ct_prj_src_chanid_fk
+            REFERENCES rhnChannel(id)
+            ON DELETE CASCADE,
+
+    config_channel_id NUMBER
+        CONSTRAINT suse_ct_prj_src_confchanid_fk
+            REFERENCES rhnConfigChannel(id)
+            ON DELETE CASCADE
```

Comment:
Same as on EnvironmentTarget. AFAIK we do not want to implement this in the first shot. So I think we should not add it to code or DB schema